README.md

```diff
@@ -220,6 +220,7 @@ A standalone string cannot be modified, i.e. `data = 'a'; ajv.validate(schema, d
 - `toLowerCase`: convert to lower case
 - `toUpperCase`: convert to upper case
 - `toEnumCase`: change string case to be equal to one of `enum` values in the schema
+- `normalize`: replace inner multiple spaces with a single space
```

Comment:
The name is too generic for what it does... Possible alternative - `trimInner` (and should it not replace tabs too?).

Any better ideas?

Reply:
word `trim` means to remove space, this just removes extra spaces, its not a bad idea though to also include remove spaces completely, with `trimInner` and come up with a better name for this one like `normalizeSpaces`

Reply:
let's please not conflate two different things into on PR.

I don't know the use case for removing all inner spaces tbh, normalizeSpaces is ok though.

Reply:
ok, will separate the other one in a different PR, will make another commit today

Reply:
review my latest commit